src/clj/game/core/diffs.clj

```diff
@@ -49,6 +49,7 @@
              :source-type :ability
              :source-info {:ability-idx ability-idx}}]
     (if (and (or (active? card)
+                 (condition-counter? card)
```

Comment:
should this check be in `active?`? Seems like it. Might not be worth fixing in this PR, tho.